llmfoundry/optim/lion8b.py

```diff
@@ -0,0 +1,417 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Callable, Dict, Iterable, Optional, Tuple
+
+import torch
+
+_KEY_MOMENTUM = 'exp_avg'
```

Comment:
just because its what pytorch does, I'd skip the keys and just use the names throughout (eg [adam](https://github.com/pytorch/pytorch/blob/main/torch/optim/adam.py))